doc/nrf/security/ap_protect.rst

```diff
@@ -197,16 +216,98 @@ This is the default setting in the |NCS|.
 
 You can start debugging the firmware without additional steps needed.
 
-Once you are done debugging, run the following command to enable the access port protection:
+.. _app_approtect_uicr_approtect:
+
+Enabling AP-Protect by locking the UICR.APPROTECT register
+==========================================================
+
+For the devices that are in a production environment, you are highly recommended to lock the UICR.APPROTECT register to prevent unauthorized access to the device.
+If the access port protection is configured this way, it cannot be disabled without erasing the flash memory.
+
+.. note::
+    This is the only mechanism supported for the nRF52 Series and the nRF9160 devices that do not have software support for AP-Protect.
```

Comment:
I dont understand what this note is trying to tell me?
Can you clarify in this pr_comment then maybe we can find some more clear formulation

Reply:
Some of the nRF52 series and nRF9160 only support the UICR.APPROTECT for locking the debugging.

Others have registers such as:
APPROTECT.DISABLE, 
SECUREAPPROTECT.DISABLE, 
APPROTECT.FORCEPROTECTS 
SECUREAPPROTECT.FORCEPROTECT
(depending on the device)

These registers allow the debugging to be disabled by the above Kconfigs, without locking the UICR.APPROTECT.

Reply:
Aha, like that. I forgot that the difference was "Hardware" and "Hardware and Software". While I should have remembered, I guess this simulates a user. How about this, to reuse the exact same wording as earlier on the page?
```suggestion
    This is the only mechanism supported for the nRF52 Series and the nRF9160 devices that do not support "Hardware and software" for AP-Protect.
```

Reply:
Added with a slight change.